tests/perf/main/install-many-snaps/task.yaml

```diff
@@ -62,7 +62,11 @@ execute: |
                 echo "already $NUM_SNAPS installed, now check other features"
                 break
             fi
-        done        
+        done
+    done
+
+    while snap changes | MATCH " (Do |Doing ).*Install \".*\" snap"; do
```

Comment:
btw. I needed this a couple of times already. Perhaps we should have something like `snap debug changes --settle`

Reply:
agree, it would be really nice to have it